book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative('./rental')
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] to make the associations work *Getters and Setters for @rentals should be present.